rpm/generic/zfs.spec.in

```diff
@@ -363,12 +385,25 @@ systemctl --system daemon-reload >/dev/null || true
 %endif
 
 %files
+# Core utilities
 %{_sbindir}/*
-%{_bindir}/*
-%{_libexecdir}/%{name}
+%{_bindir}/raidz_test
+%{_bindir}/zgenhostid
+# Optional Python 2/3 scripts
+%{_bindir}/arc_summary
+%{_bindir}/arcstat
+%{_bindir}/dbufstat
+%if 0%{?__python_pkg_version} == 3 || 0%{?__pythong_pkg_version} >= 34
```

Comment:
Typo here, I think you mean `%__python_pkg_version`.

Reply:
Good catch, fixed.